src/App.js

```diff
@@ -40,6 +40,9 @@ class App extends Component {
         forwardReadColors: "reds",
         reverseReadColors: "blues",
         colorReadsByMappingQuality: false,
```

Comment:
These shouldn't be used anymore right? We definitely don't want to duplicate these and the structured data, even if it means we have to do a bunch of `[]`-indexing in the React code for the color UI to get where we have hidden the values now.